app/src/main/res/values/strings.xml

```diff
@@ -244,7 +244,8 @@
     <!-- Menu option in the search selector menu to open the search settings -->
     <string name="search_settings_menu_item">Search settings</string>
     <!-- Header text for the search selector menu -->
-    <string name="search_header_menu_item">This time search:</string>
+    <string name="search_header_menu_item" moz:RemovedIn="109" tools:ignore="UnusedResources">This time search:</string>
+    <string name="search_header_menu_item_2">This time search in:</string>
```

Comment:
Every string needs a l10n note so that translator knows what the string is for. You can copy over the comment above.